quantization/src/encoded_vectors_binary.rs

```diff
@@ -22,6 +24,7 @@ pub struct EncodedBinVector {
 #[derive(Serialize, Deserialize)]
 struct Metadata {
     vector_parameters: VectorParameters,
+    alpha: f32,
```

Comment:
this probably should be optional, so previous versions could be loaded as well

Reply:
I have changed it to `Option<f32>` without adding a test case. Should there be added storage compatibility test cases on qdrant side?

Reply:
We might want `#[serde(default)]` before it as well.